package.json

```diff
@@ -24,10 +24,11 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "jest",
+    "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint '**/*.{js,jsx}'",
-    "pretty": "prettier --write \"./**/*.{js,jsx}\""
+    "pretty": "prettier --write \"./**/*.{js,jsx}\"",
+    "fix": "!src/**/*/*.snap"
```

Comment:
this is not correct. you should combine yarn run pretty and yarn run fix into one command. please work with Abdul rahman or ammar or ask halit